### PR TITLE
wallet-cli/rpc: allow sweep_all to use outputs in all subaddresses wi…

### PR DESCRIPTION
**Pull request by stoffu**
…thin an account

### CODE REVIEW

File: src/wallet/wallet_rpc_server.cpp [hunk 1]
```diff
@@ -1394,11 +1394,22 @@ namespace tools
       return  false;
     }
 
+    std::set<uint32_t> subaddr_indices;
+    if (req.subaddr_indices_all)
+    {
+      for (uint32_t i = 0; i < m_wallert->get_num_subaddresses(req.account_index); ++i)
```
Comment (moneromooo-monero) [inline src/wallet/wallet_rpc_server.cpp]: m_wallert

Comment (stoffu) [reply src/wallet/wallet_rpc_server.cpp]: oops :p

**Comment by teran-mckinney** [comment]
Merry Christmas!

I'm wondering if this is merge ready or if anyone else needs to review it.

**Comment by teran-mckinney** [comment]
Thank you!

**Pull request merged**